plugins/hydra_joblib_launcher/hydra_plugins/hydra_joblib_launcher/_core.py

```diff
@@ -50,6 +50,18 @@ def execute_job(
     return ret
 
 
+def process_joblib_cfg(joblib_cfg: Dict[str, Any]) -> Dict[str, Any]:
+    for k, v in joblib_cfg.items():
+        if k in ["pre_dispatch", "batch_size", "max_nbytes"] and v:
+            try:
+                joblib_cfg[k] = int(v)
```

Comment:
Wouldnt this convert the floats to int as well?

Reply:
these three params only allows int (https://joblib.readthedocs.io/en/latest/parallel.html#parallel-reference-documentation), so I think we should be good here?